### ISSUE TITLE
I'm very interested to see the comparison with densitycut: https://bitbucket.org/jerry00/densitycut_dev

### ISSUE THREAD

**Issue by JerryDing** [lmweber/cytometry-clustering-comparison#1]

**Comment by lmweber** [comment]
Hi Jerry, thank you for bringing this new clustering algorithm to our attention. We will try it out on our data sets and aim to add an additional set of results on this website. Best regards, Lukas

**Comment by JerryDing** [comment]
Hi Lukas, thank you in advance for putting the effort to do the analysis. Looking forward to seeing the new results! Cheers, Jerry

**Comment by lmweber** [comment]
Hi Jerry, I have added a report with additional results for densityCut, saved in the updates/densityCut/ directory. Sorry it took a while. It's a nice algorithm, and performance was intermediate overall, according to our evaluations. In particular, we note that densityCut can be used for lots of biological data types, not just CyTOF. However, there were some important differences in our evaluation methodologies, which affected the results; I have explained these in the report. I hope the additional results are useful, and let us know if you have any questions. Best regards, Lukas
